docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1220,6 +1220,16 @@ How to **investigate**
 - Flush tenant's data to blocks storage.
 - Remove tenant's directory on disk and restart ingester.
 
+## MimirStoreGatewayTooManyFailedOperations
+
+How it **works**:
+
+This alert fires when the `store-gateways` report errors when interacting with the object storage.
+
+How to **investigate**
+
+- Check the `store-gateways` logs
```

Comment:
Can we provide an example of the log line to look for? I assume the actual message will be specific to each object storage backend but there should be something we can add. Maybe a LogQL query too? I'd also mention the `Mimir / Object Store` dashboard.

Reply:
I'm not using loki in production so my suggested query might not be accurate.
I've included a log sample as well.